subsys/bluetooth/host/conn.c

```diff
@@ -3674,6 +3674,22 @@ int bt_conn_le_create(const bt_addr_le_t *peer, const struct bt_conn_le_create_p
 	struct bt_conn *conn;
 	int err;
 
+	CHECKIF(ret_conn == NULL) {
+		return -EINVAL;
+	}
+
+	CHECKIF(*ret_conn != NULL) {
+		/** This warning is to inform the user that the connection reference is not NULL and
+		 * it may lead to a potential connection object leakage.
+		 */
+		LOG_WRN("*conn should be unreferenced and initialized to NULL before passing to"
+			" %s", __func__);
```

Comment:
Seems overkill to add the function name, when it's already added if `CONFIG_LOG_FUNC_NAME_PREFIX_WRN=y`
```suggestion
		LOG_WRN("*conn should be unreferenced and initialized to NULL");
```